bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/handler/ControllerHandler.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.doorbird.internal.handler;
+
+import static org.openhab.binding.doorbird.internal.DoorbirdBindingConstants.*;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.doorbird.internal.api.DoorbirdAPI;
+import org.openhab.binding.doorbird.internal.api.DoorbirdInfo;
+import org.openhab.binding.doorbird.internal.config.ControllerConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ControllerHandler} is responsible for handling commands
+ * to the A1081 Controller.
+ *
+ * @author Mark Hilbush - Initial contribution
+ */
+@NonNullByDefault
+public class ControllerHandler extends BaseThingHandler {
+    private final Logger logger = LoggerFactory.getLogger(ControllerHandler.class);
+
+    private @Nullable String controllerId;
+
+    private DoorbirdAPI api = new DoorbirdAPI();
+
+    public ControllerHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        String host = getConfigAs(ControllerConfiguration.class).doorbirdHost;
```

Comment:
Why don't you get the config once and re-use it?

Reply:
Done

Reply:
was this addressed?

Reply:
I thought I had addressed it in both handler classes, but I must've missed it here. Fixed now.